src/resources/public/js/script.js

```diff
@@ -109,6 +109,26 @@ function submitURL(requestData) {
     });
 }
 
+function onSignIn(googleUser) {
+  var id_token = googleUser.getAuthResponse().id_token;
+  var xhr = new XMLHttpRequest();
+  xhr.open('POST', '/a/api/auth/signin');
+  xhr.setRequestHeader('Content-Type', 'application/json');
+  xhr.send(JSON.stringify({
+    tokenId: id_token,
+  }));
+}
```

Comment:
- You have to handle response data after posting it. 
example response: 
{
    email: user@gmail.com, ​
   ​ accessToken: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
}
- Use *email* to show on screen ( depend on the design ). Save the *accessToken* in the cookie for future use
- Handle error in case fail to post or invalid tokenId

Reply:
Use jQuery AJAX or Axios
https://api.jquery.com/jquery.ajax/